Source_Code/Client/ClueLessClient.java

```diff
@@ -260,6 +264,10 @@ public static void main(String[] args)
     private final int ServerPort;
 
     private final Player UserPlayer;
+    
+    private ConsoleInput reader = new ConsoleInput();
```

Comment:
I needed to initialize this outside of main in order for the processMsg methods to be able to read from the console.